### PR TITLE
STDEV-5035 updated gradle-wrapper to 6.1.1

### PR DESCRIPTION
**Pull request by castasat**
fixes build error: "The current Gradle version 4.6 is not compatible with the Kotlin Gradle plugin"

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -3,7 +3,8 @@ apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 
 apply plugin: 'maven-publish'
-apply plugin: 'org.jetbrains.dokka-android'
+// TODO
+// apply plugin: 'org.jetbrains.dokka-android'
```
Comment (xcam) [inline build.gradle]: ?

File: build.gradle [hunk 2]
```diff
@@ -59,7 +60,8 @@ buildscript {
     dependencies {
         classpath 'com.android.tools.build:gradle:3.2.1'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-        classpath "org.jetbrains.dokka:dokka-android-gradle-plugin:0.9.18"
+        // TODO
+        // classpath "org.jetbrains.dokka:dokka-android-gradle-plugin:0.9.18"
```
Comment (xcam) [inline build.gradle]: ?

File: build.gradle (hunk 1 above)
Comment (castasat) [reply build.gradle]: восстановил доку

File: build.gradle (hunk 2 above)
Comment (castasat) [reply build.gradle]: восстановил доку

**Review by xcam — APPROVED** [review]

**Pull request merged**